binaries/cli/src/command/inspect/top.rs

```diff
@@ -0,0 +1,421 @@
+    refresh_duration: Duration,
+) -> eyre::Result<()> {
+    let mut app = App::new();
+    let mut last_update = Instant::now();
+    let mut last_node_query = Instant::now();
+    let mut system = System::new_all();
+    let mut node_infos: Vec<NodeInfo> = Vec::new();
+
+    // Reuse coordinator connection
+    let mut session = connect_to_coordinator((coordinator_addr, coordinator_port).into())
+        .wrap_err("Failed to connect to coordinator")?;
+
+    // Query node info once initially
+    let request = ControlRequest::GetNodeInfo;
+    let reply_raw = session
+        .request(&serde_json::to_vec(&request).unwrap())
+        .wrap_err("failed to send initial request to coordinator")?;
+
+    let reply: ControlRequestReply =
+        serde_json::from_slice(&reply_raw).wrap_err("failed to parse reply")?;
+
+    node_infos = match reply {
+        ControlRequestReply::NodeInfoList(infos) => infos,
+        ControlRequestReply::Error(err) => {
+            return Err(eyre!("coordinator error: {err}"));
+        }
+        _ => {
+            return Err(eyre!("unexpected reply from coordinator"));
+        }
+    };
+
+    loop {
+        terminal.draw(|f| ui(f, &mut app, refresh_duration))?;
+
+        let timeout = refresh_duration
+            .checked_sub(last_update.elapsed())
+            .unwrap_or(Duration::from_millis(100));
+
+        if event::poll(timeout)? {
+            if let Event::Key(key) = event::read()? {
+                if key.kind == KeyEventKind::Press {
+                    match key.code {
+                        KeyCode::Char('q') | KeyCode::Esc => {
+                            return Ok(());
+                        }
+                        KeyCode::Down | KeyCode::Char('j') => {
+                            app.next();
+                        }
+                        KeyCode::Up | KeyCode::Char('k') => {
+                            app.previous();
+                        }
+                        KeyCode::Char('n') => {
+                            app.toggle_sort(SortColumn::Node);
+                        }
+                        KeyCode::Char('c') => {
+                            app.toggle_sort(SortColumn::Cpu);
+                        }
+                        KeyCode::Char('m') => {
+                            app.toggle_sort(SortColumn::Memory);
+                        }
+                        KeyCode::Char('r') => {
+                            // Force refresh node list
+                            last_node_query = Instant::now()
+                                .checked_sub(Duration::from_secs(60))
+                                .unwrap_or(Instant::now());
+                        }
+                        _ => {}
+                    }
+                }
+            }
+        }
+
+        // Update data if refresh interval has passed
+        if last_update.elapsed() >= refresh_duration {
+            // Refresh system info (process metrics)
+            system.refresh_all();
```

Comment:
Looks like this isn't used anymore. I think we can remove `sysinfo` completely from this module.